solr/core/src/java/org/apache/solr/update/TransactionLog.java

```diff
@@ -102,6 +102,9 @@ public Object resolve(Object o, JavaBinCodec codec) throws IOException {
   };
 
   public class LogCodec extends JavaBinCodec {
+
+    private static final byte UUID = 20;
```

Comment:
There is a risk that somebody may accidently use the code `20` and break `LogCodec`. So, let's keep this static field in JavabinCodec and say "This is reserved to be used only in LogCodec"